lib/slackbot.rb

```diff
@@ -0,0 +1,28 @@
+require 'slack-ruby-bot'
+require_relative './darksky.rb'
+require_relative './google_coordinates.rb'
+
+class Slackbot < SlackRubyBot::Bot
+  command 'weather now' do |client, data, match|
+    location = match[:expression] || 'Washington DC'
+    location_description, coordinates = GoogleCoordinates.location_info(location)
```

Comment:
I like this, but it took me a minute to parse. Using destructuring to determine if your message is a location or an error message is pretty clever. I might rename the variable you store it in and comment as to the dual nature of the value.